### DISCUSSION TITLE
prisma doesn't migrate in docker container on arm m1

### DISCUSSION THREAD

**Discussion by lecheynata** [prisma/prisma#13300] [Q&A]
I have a problem when trying to generate the migration inside a container with mariadb running. The database server blocks connections when running the task with the following message "[Warning] Aborted connection 7 to db: 'db_material' user: 'root' host: '172.18.0.3' (Got an error reading communication packets)".

<img width="1112" alt="Screen Shot 2022-05-11 at 11 20 04 a m" src="https://user-images.githubusercontent.com/23022461/167898934-3e30175d-c2a2-4f1e-9341-a58e1818ca6f.png">

This does not happen when replicating the project within the amd64 architecture.

Has anyone had the same problem?

**Comment by austincrim** [comment]
Hey @lecheynata 👋 ,

What commands specifically are you running when you get this error?

Can you share the relevant Docker config?

**Reply by lecheynata** [reply]
hi! @austincrim, sure.

At the moment of executing the command `yarn prisma migrate dev`, is when the database server throws the error, but if I execute `yarn prisma db seed` it executes it without problems and it does not show me any error.

This is the [link](https://github.com/lecheynata/example-microservices) to my repository with the configuration I use to build my project.

https://user-images.githubusercontent.com/23022461/167959112-7bf8b452-9867-4a77-a71a-a1af29afba14.mp4

**Reply by austincrim** [reply]
Thanks for the repo! I was able to recreate this issue as well. The fact that it works with `seed` but not with `migrate` makes me think it's an issue related to the [shadow database](https://www.prisma.io/docs/concepts/components/prisma-migrate/shadow-database).

Would you mind opening a [bug report](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) with a link to your repo?

**Reply by lecheynata** [reply]
Of course, thank you very much for your time.

**Reply by lecheynata** [reply]
hi again @austincrim I wrote the issue but they just deleted it

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
